### PR TITLE
Walking over blood makes you slip

### PR DESCRIPTION
**Pull request by SteelSlayer**
## What Does This PR Do
Nanotrasen put something in the water that made everyone's blood slippery. I sure hope the janitors aboard the Cyberiad are well-trained.

## Why It's Good For The Game
This is very important for realism and immersion. Don't you know puddles of blood are slipping hazards?

## Images of changes
e

## Changelog
:cl:
add: Walking over non-dried blood makes you slip
/:cl:

### CODE REVIEW

**Comment by MarsM0nd** [comment]
I guess everyone will need to switch to wallking most of the time, rather the other way around we currently have.

**Comment by BakKeener** [comment]
Just a suggestion but perhaps make it RNG based on how much blood is there? For example, drops blood you see around the station, you probably won't slip on them despite running all over the place and it would be silly running through maint chasing an antag that is bleeding and isn't a slime to only slip on drops of blood. Trails of blood should have a low chance to slip on while puddles of the stuff are our real big slippers. You also have to keep in mind that blood doesn't cover the entire tile like water and lube does *at least visually speaking* of course.

**Comment by AffectedArc07** [comment]
Big no from me, bubblegum is already impossible to kill. This will make it require no slip shoes for you to even have a chance.

**Comment by RaidarV** [comment]
ogod no, please no.

**Comment by Drobe50** [comment]
I'm gonna love it when an assistant is dragging a monkey around beating it to death while leaving a trail of blood from arrivals to escape.

**Comment by MRgaty** [comment]
I wanna die, this is so bad but at the same time looks so hilariously chaotic

**Comment by datlo** [comment]
You're about a week too early for april fools PR :eyes:

**Comment by Gatchapod** [comment]
While you're at it, add stamina that limits the time you can spend running, Bay style.
The last step will be adding "Running in the Hallways" as a crime to the Space Law.

**Comment by Carthusia** [comment]
Going to agree, this is a big no from me.

You would basically turn everyone into slimepeople in how their blood works.

We've all seen the chaos of a bleeding slime person in medbay, add this, and you've multiplied that by 8x in any given emergency.

Medical staff slipping while trying to provide life saving care, security slipping because the guy their after is bleeding, or the guy their chasing is pulling a bleeding body and people just being murdered by antags because the halls are filled with blood and now they cant escape.

It would be horrible.

File: code/game/objects/effects/decals/Cleanable/humans.dm [hunk 1]
```diff
@@ -61,6 +63,12 @@ GLOBAL_LIST_EMPTY(splatter_cache)
 		deltimer(dry_timer)
 	return ..()
 
+/obj/effect/decal/cleanable/blood/proc/BloodySlip(datum/source, mob/living/carbon/human/H)
+	if(!istype(H))
+		return
+	var/mob/living/carbon/human/slipper = H
+	slipper.slip("the blood", 3, 2, tilesSlipped = 0, walkSafely = 1)
```
Comment (farie82) [inline code/game/objects/effects/decals/Cleanable/humans.dm]: walkSafely is a boolean value. So `TRUE` here

**Comment by Spacemanspark** [comment]
Halt

**Comment by Mitchs98** [comment]
Yeah...I just...why?

File: code/game/objects/effects/decals/Cleanable/humans.dm (hunk 1 above)
Comment (SteelSlayer) [reply code/game/objects/effects/decals/Cleanable/humans.dm]: I could make a slippery component, and be all modern and stuff, but screw that! We need more oldschool code, more copy and paste, more 1s and 0s, down with TRUE and FALSE! They are fake variables to hide their true meaning anyway.

**Comment by taukausanake** [comment]
Give it a low percentage chance of slip instead of outright slip. I don't want cutting yourself when you're on the run from security to be a viable strategy

**Comment by Ty-Omaha** [comment]
add this in on april fools

**Comment by farie82** [comment]
I don't like this change in all honesty. But adding random chance to this will even make it worse. Random chance in these situations will only lead to frustration. Especially small chances. That's bad game design.

**Comment by SteelSlayer** [comment]
So I should add a 1% chance to slipping on blood?

**Comment by farie82** [comment]
> So I should add a 1% chance to slip on blood?

At a 1% chance I'll go and travel to your city then :gun: 🦆

**Comment by Carthusia** [comment]
> So I should add a 1% chance to slip on blood?

0% chance

Blood slipping will cause far to many issues.

Example: See any situation where a slime is bleeding and people are around

**Comment by Ty-Omaha** [comment]
Keep it at 100% this is a perfect PR for april fools

**Comment by VEN0M5** [comment]
I can already imagine the horror of T Spiders with this. Maybe add this as a feature to be enabled by admins for shenanigans.

Anyways, a big no as a regular thing.

**Comment by MRgaty** [comment]
I dont know what are you talking about guys but this seems to be pretty balanced

**Comment by Denthamos** [comment]
I could see maybe slipping on undried gib piles, but not blood.

**Comment by daylight2** [comment]
This would be giving a massive advantage to defenders in combat. Double-esword wielding cling shot a revolver at you? Just run away, and laugh as they either slip or chase you at walking speed, whilst you have no movement penalty other than the damage that caused you to bleed. And adding a % chance to slip is just god awful, as I'm sure getting fucked over by RNG in our stun-based combat economy will feel satisfying and wholesome to all involved.

**Comment by SteelSlayer** [comment]
This was never a serious PR, don't worry. Something something April fools.

**Pull request closed without merging**